menoh/tensorrt/Inference.cpp

```diff
@@ -321,19 +324,28 @@ namespace menoh_impl {
 
                 CHECK(cudaStreamSynchronize(stream));
                 CHECK(cudaStreamDestroy(stream));
-            }
+            };
 
-#ifdef MENOH_ENABLE_TENSORRT_PRIFILER
-            auto end = clock::now();
-            std::cout << "Run time = "
-                      << std::chrono::duration_cast<std::chrono::milliseconds>(
-                           end - start)
-                             .count() /
-                           1000.0
-                      << " sec" << std::endl;
-
-            gProfiler.printLayerTimes();
-            std::cout << "Inference::Run::done" << std::endl;
+#ifdef MENOH_ENABLE_TENSORRT_PROFILER
+            if(config_.enable_profiler) {
+                std::cout << "Inference::Run::start" << std::endl;
+                using clock = std::chrono::high_resolution_clock;
+                auto start = clock::now();
+                runner();
+                auto end = clock::now();
+                std::cout
+                  << "Run time = "
+                  << std::chrono::duration_cast<std::chrono::milliseconds>(
+                       end - start)
+                         .count() /
+                       1000.0
+                  << " sec" << std::endl;
+
+                gProfiler.printLayerTimes();
+                std::cout << "Inference::Run::done" << std::endl;
+            } else {
+                runner();
+            }
 #endif
```

Comment:
If `MENOH_ENABLE_TENSORRT_PROFILER` is not defined `runner` will not be executed.
Is that intended behavior?

Reply:
No. I fixed it #201. I'll merge it to this PR.